.github/workflows/test-suite.yml

```diff
@@ -1,6 +1,6 @@
 name: 'Test Suite'
```

Comment:
For the PR title remember that commits are categorised from a users point of view. This change would not be a new provider feature but more of an internal chore which enhances our testing.

Reply:
You are right! I changed the PR title to chore